_content/lorawan/lorawan_specs.md

```diff
@@ -0,0 +1,20 @@
+---
+title: LoRaWAN Specifications
+---
+
+# LoRaWAN Specifications
+
+The LoRaWAN™ specification documents are developed and maintained by the LoRa Alliance: an open association of collaborating members. To fully define the LoRaWAN protocol, and to ensure interoperability among devices and networks, the LoRa Alliance develops and maintains the following documents:
```

Comment:
This is copied straight from their website, right? I'm not sure if we should do that.